1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,176 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @xxx
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        # Save button is removed as Project API demands a PM
+        And they see a Next Button
+
+    @xxx
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard"
+        # but this one enables a Curator to edit projects where he's not an
+        # participant (as an ordinary User)
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @xxx
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Button Add Project Participant
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Button "<Button>" decided by Project's "<Status>"
+        And they see a Next Button
+        Examples:
+            | Role                  | Status    | Button                   |
+            | Curator               | Published | Update published project |
+            | Curator               | Draft     | Save draft               |
+            | Project Owner         | Published | Update published project |
+            | Project Owner         | Draft     | Save draft               |
+            | Project Manager       | Published | Update published project |
+            | Project Manager       | Draft     | Save draft               |
+            | Local Project Manager | Published | Update published project |
+            | Local Project Manager | Draft     | Save draft               |
```

Comment:
I praksis holder det vel med 2 eksempel her?
```
| Status      | Button                             |
| Draft        | Save draft                        |
| Published | Update published project |
```

så kan heller rollene flyttes til Given-And:
```
And User has one of these role in the project:
  | Curator |
  | Project Owner |
  (...)
```

Reply:
Takk! Det hjalp.